diff --git a/node_modules/osu-parsers/lib/node.cjs b/node_modules/osu-parsers/lib/node.cjs
index 5b7af2d..8191495 100644
--- a/node_modules/osu-parsers/lib/node.cjs
+++ b/node_modules/osu-parsers/lib/node.cjs
@@ -1690,6 +1690,8 @@ class BeatmapHitObjectDecoder {
     hitObject.hitSound = Parsing.parseInt(data[4]);
 
     const bankInfo = new osuClasses.SampleBank();
+    bankInfo.normalSet = osuClasses.SampleSet.None;
+    bankInfo.additionSet = osuClasses.SampleSet.None;
 
     this.addExtras(data.slice(5), hitObject, bankInfo, offset, beatmap.fileFormat);
     this.addComboOffset(hitObject, beatmap);
diff --git a/node_modules/osu-parsers/lib/node.mjs b/node_modules/osu-parsers/lib/node.mjs
index f29e091..c63bf48 100644
--- a/node_modules/osu-parsers/lib/node.mjs
+++ b/node_modules/osu-parsers/lib/node.mjs
@@ -1686,6 +1686,8 @@ class BeatmapHitObjectDecoder {
     hitObject.hitSound = Parsing.parseInt(data[4]);
 
     const bankInfo = new SampleBank();
+    bankInfo.normalSet = osuClasses.SampleSet.None;
+    bankInfo.additionSet = osuClasses.SampleSet.None;
 
     this.addExtras(data.slice(5), hitObject, bankInfo, offset, beatmap.fileFormat);
     this.addComboOffset(hitObject, beatmap);
